composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/chat/CurrentChatThreadScreen.kt

```diff
@@ -71,7 +71,7 @@ class CurrentChatThreadScreen(private val chatThread: ChatThread) : Screen {
 
         Scaffold(
             topBar = {
-                TopAppBar(chatThread.name, false)
+                TopAppBar(chatThread.name, true, GroupChatEditScreen(chatThread))
```

Comment:
Jest to trochę nieczytelne, przed `true` musi być na pewno `showManagmentButton: `.
Oraz tak jak mówiłem, onClick ale tak nazwany aby było wiadomo po wciśnięciu jakiego elementu uruchomi się dana funkcja anonimowa przekazana do niej.